tests/integration/test_parameterisations.py

```diff
@@ -147,6 +157,16 @@ def spm_two_signal_cost(self, parameters, model, x0):
     )
     @pytest.mark.integration
     def test_multiple_signals(self, optimiser, spm_two_signal_cost, x0):
+        # Some optimisers require a complete set of bounds
+        if optimiser in [pybop.SciPyDifferentialEvolution]:
```

Comment:
Does the below represent what we expect end-users to do? It seems like it could be pushed into the codebase and abstracted from the user if so.

Reply:
No, this should be updated when we move to using a Parameters object instead of a list of parameters! Here, it is written this way in order to re-use the cost fixture.